source/Engine.cpp

```diff
@@ -1539,9 +1539,10 @@ void Engine::CalculateStep()
 // boarding events, fire weapons, and launch fighters.
 void Engine::MoveShip(const shared_ptr<Ship> &ship)
 {
-	// Various actions a ship could have taken last frame may have impacted its jump capabilities.
-	// Therefore, recalibrate its jump navigation information.
-	ship->RecalibrateJumpNavigation();
+	// Various actions a ship could have taken last frame
+	// may have impacted its jump and fighting capabilities.
+	// Therefore, recalibrate all cached information.
```

Comment:
Let's have this be agnostic to what we're caching so that it doesn't need updated if we cache new stuff in the future.
```suggestion
	// Various actions a ship could have taken last frame may have impacted the accuracy of cached values.
	// Therefore, determine with any information needs recalculated and cache it.
```